@here/harp-datasource-protocol/lib/TileInfo.ts

```diff
@@ -92,6 +92,23 @@ export class FeatureGroup {
             this.typeIndex.length = startSize;
         }
     }
+
+    /**
+     * Compute size in bytes.
+     */
+    getNumBytes(): number {
+        return (
+            (this.featureIds.length +
+                this.techniqueIndex.length +
+                this.textIndex.length +
+                this.positionIndex.length +
+                this.positions.length +
+                (this.layerIndex !== undefined ? this.layerIndex.length : 0) +
+                (this.classIndex !== undefined ? this.classIndex.length : 0) +
+                (this.typeIndex !== undefined ? this.typeIndex.length : 0)) *
+            4
```

Comment:
```suggestion
            8
```
Should be 8, since all ``number`` in javascript are actually doubles.

Reply:
Done

Reply:
Done